### DISCUSSION TITLE
Make font "Segoe UI Variable" not "Segoe UI"

### DISCUSSION THREAD

**Discussion by Posandu** [blueedgetechno/win11React#50] [Q&A]
Read the title.... Get it ?

**Comment by blueedgetechno** [comment]
Me after changing it and seeing the ui:

![](https://i.pinimg.com/originals/5c/41/6b/5c416b17a323fa6ecceb25f6c9b93ddb.gif)

**Reply by Posandu** [reply]
Can you share a screenshot ?

**Reply by blueedgetechno** [reply]
You can see that yourself. Just go to dev tool, select the body tag and change the font family.

**Reply by Posandu** [reply]
Go to this link https://aka.ms/SegoeUIVariable and download the font

**Reply by Posandu** [reply]
# Before
![image](https://user-images.githubusercontent.com/76736580/128137168-ec520242-f6b1-4c8d-be9d-5de15f8250a4.png)

# After
![image](https://user-images.githubusercontent.com/76736580/128137141-ef40f1a2-de26-4e6f-9a1d-e193536ce2b6.png)

**Reply by Posandu** [reply]
download the img first to see the changes

**Reply by blueedgetechno** [reply]
#### Changes in Brave (Chromium browser)

Without UI Variable (smooth, thin and short letters)
![withoutSegoe](https://user-images.githubusercontent.com/54899800/128170303-589ad054-7a80-4a65-a5c8-813252eb3604.png)

With UI Variable (rough curves, slightly medium bold and long letters)
![withSegoe](https://user-images.githubusercontent.com/54899800/128170870-7bd33de9-b1cc-48d6-9e0a-512ed69f000e.png)

**Reply by Posandu** [reply]
sus

**Comment by yashash-pugalia** [comment]
the reason for the rough edges with Segoe UI Variable is that `font-variation-settings: "opsz" <font-size>` was not used  opsz is short for optical sizing
Variable fonts are supposed to look better & be more legible.

![image](https://user-images.githubusercontent.com/89068816/174425098-b8246bb8-53c9-4107-9f5d-8e2403442aab.png) top win11React, bottom real win11
for the slightly bold part - the current one looks like it is has little less font-weight but with Segoe UI Variable it looks identical to real win11
